contracts/upgrade/V3VoteScript.sol

```diff
@@ -70,20 +70,50 @@ contract V3VoteScript is OmnibusBase {
             call: _forwardCall(TEMPLATE.AGENT(), TEMPLATE.LOCATOR(), abi.encodeCall(IOssifiableProxy.proxy__upgradeTo, (TEMPLATE.NEW_LOCATOR_IMPL())))
         });
 
+        // Grant APP_MANAGER_ROLE to the AGENT
+        voteItems[index++] = VoteItem({
+            description: "3. Grant APP_MANAGER_ROLE to the AGENT",
+            call: _forwardCall(
+                TEMPLATE.AGENT(),
+                TEMPLATE.ACL(),
+                abi.encodeWithSignature(
+                    "grantPermission(address,address,bytes32)",
+                    TEMPLATE.AGENT(),
+                    TEMPLATE.KERNEL(),
+                    keccak256("APP_MANAGER_ROLE")
+                )
+            )
+        });
+
         // Set Lido implementation in Kernel
         voteItems[index++] = VoteItem({
-            description: "3. Set Lido implementation in Kernel",
+            description: "4. Set Lido implementation in Kernel",
             call: _forwardCall(
                 TEMPLATE.AGENT(),
                 TEMPLATE.KERNEL(),
                 abi.encodeCall(IKernel.setApp, (IKernel(TEMPLATE.KERNEL()).APP_BASES_NAMESPACE(), params.lidoAppId, TEMPLATE.NEW_LIDO_IMPL()))
             )
         });
 
+        // Revoke APP_MANAGER_ROLE from the AGENT on Kernel ACL
```

Comment:
Do we have a policy on revoking all the roles now?

Reply:
I though we do, don't we?
And more roles were revoked on DG even